src/main/java/org/folio/rest/impl/ReindexAuthoritiesApi.java

```diff
@@ -41,7 +42,7 @@ public void getAuthorityStorageReindex(String query, int offset, int limit,
                                          Context vertxContext) {
 
     get(ReindexJobRepository.TABLE_NAME, ReindexJob.class, ReindexJobs.class,
-      query, offset, limit, okapiHeaders, vertxContext,
+      AUTHORITY_REINDEX_JOBS_QUERY, offset, limit, okapiHeaders, vertxContext,
```

Comment:
This way filtration query will be ignored.
I'd be better to remove "query" parameter from api, or compose query and your "type" filter in case "query" should be present by requirements